hazelcast-spring/src/main/resources/hazelcast-spring-5.0.xsd

```diff
@@ -4375,6 +4380,15 @@
     </xs:complexType>
 
     <xs:complexType name="hot-restart-persistence">
+        <xs:annotation>
+            <xs:documentation>
+                DEPRECATED, use 'persistence' instead.
+                Note: If either, but not both, persistence hot-restart-persistence or
```

Comment:
`but not both, persistence hot-restart-persistence or`. It seems the word persistence here is not needed?